packages/app/src/app/store/modules/server/actions.js

```diff
@@ -0,0 +1,20 @@
+import { dispatch } from 'codesandbox-api';
+
+export function restartSandbox({ executor }) {
+  executor.emit('sandbox:restart');
+}
+
+export function restartContainer({ executor }) {
```

Comment:
@CompuIves This can be an arrow function

```suggestion
export const restartContainer = ({ executor }) => {
```